database/schema.sql

```diff
@@ -110,3 +102,15 @@ CREATE TABLE subscriber_county_assignment (
     FOREIGN KEY (subscriber_id) REFERENCES subscriber(subscriber_id),
     FOREIGN KEY (county_id) REFERENCES county(county_id)
 );
+
```

Comment:
LGTM, needed this inclusion to account for NASA image-of-the-day, moon phase images etc..